publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -117,6 +117,9 @@ private void readSpringProperties() {
                 if((rabbitmqInstanceObject.get("channelsCount") != null) ) {
                     rabbitMqProperties.setChannelsCount(Integer.parseInt(rabbitmqInstanceObject.get("channelsCount").asText()));  
                 }
+                if((rabbitmqInstanceObject.get("waitForConfirmsTimeOut") != null) ) {
+                    rabbitMqProperties.setWaitForConfirmsTimeOut(Long.parseLong(rabbitmqInstanceObject.get("waitForConfirmsTimeOut").asText()));
```

Comment:
int parseInt(String value, int defaultValue) {
  try {
    return Integer.parseInt(value);
 catch (NumbeFo... e) {
  logger.error("...", e);
  return defaultValue;
}
}

Reply:
yes changed according if we get NumberFormatException it will use default values